homeassistant/components/media_player/directv.py

```diff
@@ -327,13 +336,15 @@ def media_start_time(self):
 
     def turn_on(self):
         """Turn on the receiver."""
-        _LOGGER.debug("Turn on %s", self._name)
-        self.dtv.key_press('poweron')
+        if not self._is_client:
```

Comment:
This check should raise an error as it should never be called.

Reply:
@balloob, what kind of error should it raise? Is there something in HASS then to call for the error?

Reply:
`raise NotImplementedError()`